Automattic/jetpack
# Adds a filter for hooking when wpcom color schemes are loaded

Builds on https://github.com/Automattic/jetpack/pull/17828

#### Changes proposed in this Pull Request:

* Adds a filter for hooking when wpcom color schemes are loaded

#### Testing instructions:






* No change is expected at all follow testing instructions on https://github.com/Automattic/jetpack/pull/17828
